manifests/base/internalcert/secret.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: training-operator-webhook-secret
```

Comment:
Does it make sense to keep it in the Training Operator install similar to Katib: https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/installs/katib-standalone/kustomization.yaml#L38-L41 ?
In the future releases when we make the integration with Cert Manager for Kubeflow Install, we can remove this secret generator from Kubeflow Kustomize Overlay

Reply:
We need to keep this secret since the cert-controller doesn't create any Secret: https://github.com/open-policy-agent/cert-controller/blob/190188dbfe07022a5160380800785dd81d511726/pkg/rotator/rotator.go#L315-L318

Reply:
Yeah, I understand that. My suggest is instead of having separate folder: `/internalcert`, update the [standalone](https://github.com/kubeflow/training-operator/blob/master/manifests/overlays/standalone/kustomization.yaml) and [Kubeflow](https://github.com/kubeflow/training-operator/blob/master/manifests/overlays/kubeflow/kustomization.yaml) Kustomize installs to generate that secret as follows:
```
secretGenerator:
  - name: training-operator-webhook-secret
    options:
      disableNameSuffixHash: true
 ```

Reply:
Oh, I see.
That makes sense.

Reply:
Done.